CmdVelData:
  topic_name: /cmd_vel
  queue_size: 1000
  frequency: 10